openedx/features/personalized_learner_schedules/call_to_action.py

```diff
@@ -164,3 +178,22 @@ def _make_reset_deadlines_cta(cls, xblock, category, is_learning_mfe=False):
             }
 
         return cta_data
+
+    @classmethod
+    def _make_warn_deadlines_cta(cls, xblock, category, is_learning_mfe=False):
+        """
+        Constructs a warning call to action object containing the necessary information for the view
+        """
+        cta_data = {
+            "description": Text("{b_open}{header}{b_close} {explanation}").format(
+                b_open=HTML("<b>"),
+                b_close=HTML("</b>"),
+                form_values={},
+                header=_("It looks like you missed some important deadlines based on our suggested schedule."),
+                explanation=_(
+                    "Unfortunately you can't update this schedule and shift the past due "
+                    "assignments into the future. Contact your course team for  more information."
```

Comment:
Note: we still need to clarify the exact wording to use here. However, we definitely don't want to include a phrase like "contact your course team".

Reply:
Yep. Who can help us with that?

Reply:
I pushed 40bfce2841c5ea63df2b39dd35a147f728ea2a88. I'm checking with the client about this.